content/millennium/r4/request-and-response/communication.md

```diff
@@ -0,0 +1,129 @@
+* [In Response To](https://hl7.org/FHIR/communication-definitions.html#Communication.inResponseTo)
+* [Status (Completed)](https://hl7.org/FHIR/communication-definitions.html#Communication.status)
+* [Communication (Category Message, Reminder, Coding Query)](https://hl7.org/FHIR/communication-definitions.html#Communication.category)
+* [Priority](https://hl7.org/FHIR/communication-definitions.html#Communication.priority)
+* [Subject](https://hl7.org/FHIR/communication-definitions.html#Communication.subject)
+* [Topic](https://hl7.org/FHIR/communication-definitions.html#Communication.topic)
+* [Encounter](https://hl7.org/FHIR/communication-definitions.html#Communication.encounter)
+* [Recieved](https://hl7.org/FHIR/communication-definitions.html#Communication.received)
+* [Recipient](https://hl7.org/FHIR/communication-definitions.html#Communication.recipient)
+* [Sent](https://hl7.org/FHIR/communication-definitions.html#Communication.sent)
+* [Sender](https://hl7.org/FHIR/communication-definitions.html#Communication.sender)
+* [Payload](https://hl7.org/FHIR/communication-definitions.html#Communication.payload)
+
+## Terminology Bindings
+
+<%= terminology_table(:communication, :r4) %>
+
+## Extensions
+
+### Custom Extensions
+
+All URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/r4/StructureDefinition/{id}`
+
+ID             | Value\[x] Type    | Description
+---------------|-------------------|----------------------------------------------------------------------------------
+`reply-to`     | [`Reference`]     | Used to direct where replies to the communications should be sent. Must be a Reference(Group)
+`email-status` | [`Code`]          | Status of the electronic communication (Pending, Opened, Deleted)
+
+
+## Search
+
+Search for Communications that meet supplied query parameters:
+
+    GET /Communication?:parameters
+
+_Implementation Notes_
+
+* The Content of a communication will be returned through the Binary Resource. A refernece to the payload will be provided in the Payload seciton.
+* All Coding Querys are also Messages. A search for the category of Messages will return Coding Queries as part of the response bundle. A search for the category of Coding Queries will return items that have both the category of Coding Query and Message.
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: false, system: true) %>
+
+### Parameters
+
+ Name         | Required? | Type          | Description
+--------------|-----------|---------------|-----------------------------------------------------------------------------------------------------------------------------------------------
+ `_id`        | This or `category` or `recipient` | [`token`]     | The logical resource id associated with the resource. Example: `C-1123_0`
+ `category`   | This or `_id` or `recipient` | [`token`] | A token for a [`CodeableConcept`] that points to the Codeable concept used for Communiation.category. 
```

Comment:
Update required

Add an example

Codeable concept -> CodeableConcept

Reply:
https://github.com/cerner/fhir.cerner.com/pull/433/commits/09eadbbe6726985d8b4f763ec3589c828180e921#diff-a46637e8cbba5ebbd797adadefc1bc2eR65

Reply:
Your example is wrong

Reply:
https://github.com/cerner/fhir.cerner.com/pull/433/commits/924ad1575295096a798f32771e85ca39f44272c6#diff-a46637e8cbba5ebbd797adadefc1bc2eR64